opta/nice_subprocess.py

```diff
@@ -60,65 +74,121 @@ def nice_run(  # type: ignore # nosec
     exit_timeout: Optional[float] = None,
     tee: bool = True,
     check: bool = False,
+    use_asyncio_nice_run=False,
     **kwargs,
 ) -> CompletedProcess:
 
-    try:
-        listargs = list(popenargs)
-        listargs[0].insert(0, "exec")
-        popenargs = tuple(listargs)
-        log_to_datadog(
-            "Calling subprocess with these arguments:\n" + " ".join(*popenargs), "INFO"
-        )
+    if (
+        use_asyncio_nice_run is False
+    ):  # Just use the original blocking Popen, this doesn't log subprocess output to datadog
```

Comment:
can this be put in the Docstring so it doesn't require to look inside the function when calling it?

Reply:
Yea we dont really have a convention around documenting functions, lets discuss that in the team and then we can add such knowledge into a standardized function docstring for example.

Reply:
ok, let's do that 👍